paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentElement.kt

```diff
@@ -62,10 +59,14 @@ internal fun PaymentElement(
         id = R.dimen.stripe_paymentsheet_outer_spacing_horizontal
     )
 
+    val selectedIndex = remember(selectedItem, supportedPaymentMethods) {
```

Comment:
looks like this is based on the payment method code now whereas before it was based on the whole payment method. Is there a reason for this change?

Reply:
This was due to the change from specs -> formElements in SupportedPaymentMethod.

form elements aren't data classes, so doing this comparison isn't possible. For example: https://github.com/stripe/stripe-android/blob/e332c0af351d2e69948976c007f8654a05510206/payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardDetailsSectionElement.kt#L12

Reply:
ahh gotcha. Thank you for the explanation!